src/PowerSimulations.jl

```diff
@@ -1,6 +1,8 @@
 isdefined(Base, :__precompile__) && __precompile__()
 module PowerSimulations
 
+using InteractiveUtils
```

Comment:
why using here and not Import? is this a left over from other stuff?

Reply:
This is a leftover from a debugging session. Resolved in f877cf4d.